topostats/plotting.py

```diff
@@ -272,9 +278,63 @@ def _set_label(self, var: str):
         self.label = self.var_to_label[var]
         LOGGER.debug(f"[plotting] self.label     : {self.label}")
 
-    def summarise_by_image(self):
-        """Summarise statistics by image."""
-        self.summary_data = self.df.groupby(["image", "threshold"]).describe()
+    @staticmethod
+    def get_relative_paths(paths: List[Path]) -> list:
```

Comment:
As this and `convert_basename_to_relative_path` are static methods I was thinking they might be useful functions in `topostats.io` sub-module since they deal with input/output paths as there may be use for them elsewhere.

What do you think?

Reply:
I was debating whether to do this, but since you flagged it, I might as well, done 👍